jupyterlab_wipp_plugin_creator/jupyterlab_wipp_plugin_creator/handlers.py

```diff
@@ -96,29 +101,118 @@ def post(self):
             template = Template(open(templatePath).read())
 
             # Generate dockerfile with user inputs, hardcoded for the time being
-            template.stream(baseImage= "python").dump(pluginOutputPath + '/Dockerfile')
+            template.stream(baseImage= "python").dump(dockerPath)
             logger.info(f"Dockerfile Template generated from jinja2 template, src/dockerfile.j2" )
 
-
         except Exception as e:
             logger.error(f"Error writing files", exc_info=e)
             self.write_error(500)
+            return
 
         # Copy files to temp location with shutil
         # Copy2 is like copy but preserves metadata
         try:
             if filepaths:
                 for filepath in filepaths:
                     filepath =  os.path.join(os.environ['HOME'], filepath)
-                    copy2(filepath, pluginOutputPath)
-                logger.info(f"Copy command completed")
+                    copy2(filepath, srcOutputPath)
+                logger.info("Copy command completed.")
             else:
                 logger.error(f"No file to copy. Please right click on file and select 'Add to new WIPP plugin'.")
+                self.write_error(500)
+                return
 
         except Exception as e:
             logger.error(f"Error when running copy command.", exc_info=e)
+            self.write_error(500)
+            return
+
+        debugEnv = os.getenv("WIPP_PLUGIN_CREATOR_DISABLE_BUILD")
+        
+        # if debugEnv was specified by the user, don't execute kubernetes commands to build the actual plugin
+        if (debugEnv):
+            logger.info("Debug mode ON. Environment is local. Plugin manifest(plugin.json) and dockerfile are generated but no images will be created. Use 'export WIPP_PLUGIN_CREATOR_DISABLE_BUILD=enterAnything' to enable full functionality if on a pod.")
```

Comment:
Please update the log output to mention "no build" mode or something like that, not a "debug mode". There might be different other options that can be granularly enabled for debugging, i.e. WIPP API calls, etc.